library/ssl_srv.c

```diff
@@ -239,9 +239,9 @@ static int ssl_parse_renegotiation_info( mbedtls_ssl_context *ssl,
  * This needs to be done at a later stage.
  *
  */
-static int ssl_parse_signature_algorithms_ext( mbedtls_ssl_context *ssl,
-                                               const unsigned char *buf,
-                                               size_t len )
+static int ssl_parse_sig_alg_ext( mbedtls_ssl_context *ssl,
```

Comment:
I am not sure, but I just saw one guide from tls1.3, which requires us to use the full name from specs.   Is the abbreviation allgin with that requirements?